swift/ql/src/queries/Security/CWE-328/WeakSensitiveDataHashing.ql

```diff
@@ -0,0 +1,57 @@
+/**
+ * @name Use of a broken or weak cryptographic hashing algorithm on sensitive data
```

Comment:
I keep wondering if we can widen the target 'on sensitive data' somehow, because I think we might miss quite a lot of good results due to this constraint.  I understand that its OK to use insecure hashes (such as MD5) for non-security purposes though, and we don't want to report those.  Also the Python version of this query seems to involve sensitive data somehow.

Reply:
Yeah, I think for example MD5 hashing is still somewhat commonplace to have a rough comparison check on files, so there might be some uses out there.